### PR TITLE
MohamedAbdi-LDN9 Lovelace-week1- Karma Clone -HTML-CSS-Module-Project/ Karma store page 🔑 -HOMEPAGE - New section

### PR DESCRIPTION
**Pull request by MohamedAbdi114**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Mohamed Abdi
- Your City:London9 lovelace
- Your Slack Name:Mohamed Abdi

# Homework Details

- Module:
- Week:HTML&CSS week1

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?
- @JDysiewicz

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,10 +11,78 @@
...
+            <a class="nav-link" href="#">How It Works</a>
+            <a class="nav-link" href="#">Store</a>
+            <a class="nav-link" href="#">Blog</a>
+            <a class="nav-link" href="#">Help</a>
+            <a class="nav-link" href="#">Login</a>
+          </div>
+        </nav>
+      </header>
+      <main>
+        <section class="Karma">
```
Comment (JDysiewicz) [inline index.html:37]: nice use of semantic `section` tags to break up the page :+1:

File: index.html [hunk 2]
```diff
@@ -10,10 +11,78 @@
...
+      </footer>
+    </body>
+  </html>
+    
+
+
+
+
+
 </body>
```
Comment (JDysiewicz) [inline index.html:87]: these closing `</body>` and `</html>` tags are redundant as you're closing them off above

**Review by JDysiewicz — APPROVED** [review]
good job, a couple small styling issues but easy fixes :P

**Comment by MohamedAbdi114** [comment]
@JDysiewicz

**Pull request closed without merging**